### ISSUE TITLE
Watch licensing issues

### ISSUE THREAD

**Issue by mikegerber** [OCR-D/ocrd_calamari#84]
A licensing issue with calamari came up:

https://github.com/Calamari-OCR/calamari/issues/3

Unfortunately I agree with @stweil here, tfaip having the GPL license requires a license change to GPL. However, it also - apparently - needs to be Apache licensed (see issue), a Gordian knot... I have high hopes in @andbue's comment about tfaip potentially switching to LGPL. (Maybe the authors aren't even aware of this issue!)

This is relevant as we would need to switch to GPL as well, if Calamari switches.

**Comment by mikegerber** [comment]
My tentative analysis is: ocrd_calamari currently depends on Calamari 1.0, which has a valid Apache license (not using tfaip and python-Levenshtein yet), so the license here is fine for the moment.

Just updating to Calamari 2 (#61) is blocked by this...

**Comment by mikegerber** [comment]
waiting for upstream

**Comment by mikegerber** [comment]
tfaip is now archived and is sticking GPL, it seems. So, it seems that Calamari 2's license will stay invalid for the time being.

Possible solutions I see:

1. Calamari could re-license to GPL while keeping the Apache parts (ocropy? kraken?) Apache - that is possible, I think. Then we could relicense to GPL for an update (if I can do this - personally I don't care much which open source license exactly, other than the obvious problem of GPL's virality, but my employer might care). 
2. Calamari removes the tfaip dependency 

(I am not a copyright lawyer.)

**Comment by stweil** [comment]
Yes. See also the [related discussion for calamari](https://github.com/Calamari-OCR/calamari/issues/3).

**Comment by mikegerber** [comment]
And of course the other contributors would have to agree, too.
